app/controllers/forest_liana/authentication_controller.rb

```diff
@@ -17,7 +17,7 @@ def initialize
     end
   
     def get_callback_url
-        URI.join(ForestLiana.application_url, "/forest/#{CALLBACK_AUTHENTICATION_ROUTE}").to_s
+      File.join(ForestLiana.application_url, "/forest/#{CALLBACK_AUTHENTICATION_ROUTE}").to_s
```

Comment:
This should work and be compatible with all OSes

```suggestion
      uri = new URI(ForestLiana.application_url)
      unless ForestLiana.application_url.end_with? '/'
        uri.path += '/'
      uri.path += "forest/#{CALLBACK_AUTHENTICATION_ROUTE}"
      uri.to_str
```